Signals/Domain/Services/Implementation/SignalsDomainService.cs

```diff
@@ -104,7 +106,13 @@ private IEnumerable<Datum<T>> FillMissingData<T>(Signal signal, TimeEnumerator t
             var missingValuePolicy = GetMissingValuePolicy(signal)
                 as MissingValuePolicy.MissingValuePolicy<T>;
 
-            return missingValuePolicy.FillMissingData(timeEnumerator, readData);
+            var olderData = missingValuePolicy.OlderDataSamplesCountNeeded > 0 
+                ? this.signalsDataRepository.GetDataOlderThan<T>(signal, timeEnumerator.FromIncludedUtc, 1)
```

Comment:
thx for catching it up. (already thinking about higher order interpolations. which I will not do. which makes this code unnecessary... :D )